README.md

```diff
@@ -74,7 +74,7 @@ By default it will look for a file called `.bumper_config.json"` but you can spe
 
 ## Development
 
-After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
+After checking out the repo, run `bin/setup` to install dependencies. Then, run `bundle exec rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
```

Comment:
This is ensure that the [specified](https://github.com/spandx/dependency_bumper/blob/76bdec4c6ae35fb65d7f19983f210a391bd1746a/Gemfile.lock#L18) version of rake is loaded and not the system rake.